README.md

```diff
@@ -19,6 +19,9 @@ An introduction to the NVIDIA Container Runtime is also covered in our [blog pos
 **If you have a custom `/etc/docker/daemon.json`, the `nvidia-docker2` package might override it.**  
 
 #### Ubuntu 14.04/16.04/18.04, Debian Jessie/Stretch
+
+The default `docker.io` package comes with Ubuntu is not supported, you need to install `docker-ce` instead. See: <https://github.com/NVIDIA/nvidia-docker/issues/887>
```

Comment:
Hello!

As mentioned in your issue, this statement is incorrect, you will need to pin the package version because docker.io doesn't install the latest version of docker

Reply:
Yes, the user have to pin the package version because the default settings will not work, that's the reason why I submit this PR.

The readme should point this out and let the user know they have to install the alternate version of docker instead of the Ubuntu default.

Could you point out which statement is incorrect in my PR for the README? I believe we need to add this notice for save other people's time.(which will include myself in the future.)

Reply:
I had changed the notice to pin the version when installing nvidia-docker2 instead of upgrade to the latest version of docker, this will be helpful too.